### DISCUSSION TITLE
闪客运行不了

### DISCUSSION THREAD

**Discussion by luqiyihui** [Gamuxorg/bbs#299] [游戏或网站运行反馈]
### 以下是推荐模板
游戏名称：闪克 Shank

操作系统：*（uname -a）*
[luqi@Archlinux ~]$ uname -a
Linux Archlinux 5.2.1-arch1-1-ARCH #1 SMP PREEMPT Sun Jul 14 14:52:52 UTC 2019 x86_64 GNU/Linux
bug描述：*（具体说说发生了什么）*
双击该游戏，没有运行起来，在终端下运行，有如下反馈：（已经安装fuse）
终端运行输出 ：*（如果有的话）*
[luqi@Archlinux Downloads]$ ./Shank.appimage 
/tmp/.mount_Shank.9eOa8k/Shank
./Shank:行7: which：未找到命令
Shank: Installed in '.'.
./Shank:行33: /tmp/.mount_Shank.9eOa8k/Shank/bin/Shank: 没有那个文件或目录
[luqi@Archlinux Downloads]$
自行尝试解决bug的方式和结果：*（如果有的话）*

**Comment by wangling12** [comment]
已更新
![2019-07-19 23-30-55 的屏幕截图](https://user-images.githubusercontent.com/24770600/61547658-68102300-aa7e-11e9-86af-1d906f0917a1.png)

**Comment by luqiyihui** [comment]
[luqi@Archlinux Downloads]$ ./Shank_i386.appimage 
/tmp/.mount_Shank_HzynK0/Shank
./Shank:行7: which：未找到命令
Shank: Installed in '.'.
./Shank:行33: /tmp/.mount_Shank_HzynK0/Shank/bin/Shank: 没有那个文件或目录
[luqi@Archlinux Downloads]$ 
问下，能帮忙解决吗？

**Comment by wangling12** [comment]
你有安装steam吗？没有的话先装个steam

**Comment by dhyanlynad** [comment]
請問你能重新上傳嗎(請問你能重新上傳嗎)
